hardware/diver/main/Core/Inc/banks/VanTaBanks.h

```diff
@@ -0,0 +1,69 @@
+#ifndef __VANTABANKS_H__
+#define __VANTABANKS_H__
+
+#include "WavePlusLUT.h"
+#include "perlin.h"
+#include <cmath>
+
+struct VanTaBanks
+{
+    DiverBankBase* PerlinStatic = new WavePlusLUT(
+        WavePlusLUT::Options{
+            .deinterlace_mode = 0,
+            .updateStyle = WavePlusLUT::UpdateStyle::Constant},
+        [](WavePlusLUT::Lookup& l, DiverUIState& state)
+        {
+            return perlin1d(float(l.i) / float(l.vres), 333, 3) * DAC_MAX_VALUE;
+        }
+    );
+
+    DiverBankBase* PerlinDynamic = new WavePlusLUT(
+        WavePlusLUT::Options{
+            .deinterlace_mode = 0,
+            .updateStyle = WavePlusLUT::UpdateStyle::PerFrame},
+        [](WavePlusLUT::Lookup& l, DiverUIState& state)
+        {
+            float pos = float(l.i) / float(l.vres);
+            float freq = (64 * state.altA_slider + 0.1) / MAX_SLIDER_VALUE;
+            int depth = int(12 * state.altB_slider / MAX_SLIDER_VALUE + 1);
```

Comment:
FYI the slider catch-up work in progress is going to turn `altA_slider` and `altB_slider` into floats.